tests/bluetooth/tester/src/audio/btp_bap_unicast.c

```diff
@@ -1528,6 +1625,23 @@ uint8_t btp_ascs_receiver_stop_ready(const void *cmd, uint16_t cmd_len,
 		return BTP_STATUS_FAILED;
 	}
 
+	if (err == 0) {
+		struct bt_conn_info conn_info;
+
+		err = bt_conn_get_info(conn, &conn_info);
+		if (err != 0) {
+			LOG_ERR("Failed to get conn info: %d", err);
+			return BTP_STATUS_FAILED;
+		}
+
+		if (conn_info.role == BT_HCI_ROLE_PERIPHERAL) {
+			/* The server the operation completes immediately */
+			btp_send_ascs_operation_completed_ev(conn, stream->ase_id,
```

Comment:
this (and few others) changes event order  (event will be sent before response to command),     is this on purpose?